src/System.Net.Security/tests/Scripts/setup-kdc.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+
+OS=`cat /etc/os-release | grep "^ID=" | sed 's/ID=//g' | sed 's/["]//g' | awk '{print $1}'`
+echo "Operating System: ${OS}"
+
+realm="TEST.COREFX.NET"
+
+principal1="HOST/host.test.corefx.net"
+principal2="HTTP"
+krb_user="krb_user"
+password="password"
```

Comment:
> but there won't be any server that would trust that ticket.

Including _this_ machine?  My concern isn't that some other machine would trust these credentials, but rather than this machine would trust them, and that, for example, after I've run this script on my Linux machine that someone else can use these wide-open credentials to gain some kind of access, even limited access, to my Linux machine.